src/ReactSVG.tsx

```diff
@@ -164,7 +171,16 @@ export class ReactSVG extends React.Component<Props, State> {
     const Wrapper = wrapper!
 
     return (
-      <Wrapper {...rest} ref={this.refCallback}>
+      <Wrapper
+        {...rest}
+        ref={this.refCallback}
+        {...(wrapper === 'svg'
```

Comment:
As per [MDN](https://developer.mozilla.org/en-US/docs/Web/SVG/Namespaces_Crash_Course#conclusion), adding all commonly used namespace declarations for SVG wrappers. This mirrors what `@tanem/svg-injector` does too.